# Holds Metadata and State for a Motor

# Name
string<=32 name

# Motor Data
float64 position
float64 velocity
float64 current

# Motor Status
int8 temperature
uint8 error_code
